.github/workflows/basic.yml

```diff
@@ -158,7 +158,9 @@ jobs:
         -DUMF_LINK_HWLOC_STATICALLY=${{matrix.link_hwloc_statically}}
 
     - name: Build UMF
-      run: cmake --build ${{env.BUILD_DIR}} -j $(nproc)
+      run: >
```

Comment:
nit:
you can use `|` instead of `>` here ([docs](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun))

and then you can omit the second `&&` and ` || ''` in the line below.

It will give a little clarity, I think. Each line would be a separate command then.

Reply:
But there is the same syntax used below:
```
    - name: Run tests
      working-directory: ${{env.BUILD_DIR}}
      run: > 
        ${{ matrix.compiler.cxx == 'icpx' && '. /opt/intel/oneapi/setvars.sh &&' || ''}}
        ctest --output-on-failure
```

so I would use the same syntax just for consistency.

Reply:
heh, we could change both places 😉

Reply:
Done

Reply:
thx!